docs/contributions.rst

```diff
@@ -281,14 +291,15 @@ Quoted from @steveno in #393:
     http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html
 
     Once you've squashed all your commits, simply do a git push -f to
-    your fork, and github will update the pull request for you
+    your fork, and GitHub will update the pull request for you
     automatically.
 
 Build script
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Add your `.c` file to somewhere in `source.mak`.
+Add your `.c` file to `source.mak`.
 
-In addition update, `win32/ctags_vs2013.vcxproj` and
-`win32/ctags_vs2013.vcxproj.filters`. So our CI process
-run on Appveyor will be failed.
+In addition, update `win32/ctags_vs2013.vcxproj` and
+`win32/ctags_vs2013.vcxproj.filters`.
+.. TODO: I'm not sure what this means -- @dkearns
+So our CI process run on Appveyor will be failed.
```

Comment:
That was mostly a note for me to learn about Appveyor.  I understood the gist of the sentence. :)